pkg/cvo/cvo_scenarios_test.go

```diff
@@ -3749,8 +3749,8 @@ func TestCVO_ParallelError(t *testing.T) {
 				{Type: DesiredReleaseAccepted, Status: configv1.ConditionTrue, Reason: "PayloadLoaded",
 					Message: "Payload loaded version=\"1.0.0-abc\" image=\"image/image:1\" architecture=\"" + architecture + "\""},
 				{Type: configv1.OperatorAvailable, Status: configv1.ConditionFalse},
-				{Type: ClusterStatusFailing, Status: configv1.ConditionTrue, Reason: "MultipleErrors", Message: "Multiple errors are preventing progress:\n* Failed to reconcile 10-a-file resource\n* Failed to reconcile 20-b-file resource"},
-				{Type: configv1.OperatorProgressing, Status: configv1.ConditionTrue, Reason: "MultipleErrors", Message: "Unable to apply 1.0.0-abc: an unknown error has occurred: MultipleErrors"},
+				{Type: ClusterStatusFailing, Status: configv1.ConditionTrue, Message: "Failed to reconcile 20-b-file resource"},
+				{Type: configv1.OperatorProgressing, Status: configv1.ConditionTrue, Reason: "MultipleErrors", Message: "Unable to apply 1.0.0-abc: multiple errors are impacting progress; see Cluster Version Operator logs for more details"},
```

Comment:
I'm not convinced about the UX of this change. In this situation, the important information is already present in the `Failing` condition - instructing the user to look into CVO logs just diverts them from the good signal/noise place (condition) to a place with more noise, but not much signal about the failure? I think that actually 

I'd like to get rid of the `MultipleErrors` here if `Failing` does not have it (because anything that does not propagate to Failing is not really an error that impacts update). I am not sure about the appropriate behavior, I think we can perhaps extract rethinking `Progressing` behavior to a new card? 

In an ideal world, I think Progressing=True should do something like this:

- Failure is single error but not a "bad" error (Failing=False): Progressing=True, Reason=failure.type, Message: failure.Message
- Failure is multiple errors but none is a "bad" error (Failing=False): Reason=Multiple (or MultipleReasons) Message: 
```
"Applying 1.0.0-abc: Observed non-problematic conditions:
* $REASON1: $MESSAGE1
* $REASON2: $MESSAGE2
```
- Failure is a single bad error (Failing=True): Probably identical to Failing? not sure.
- Failure is multiple errors, at least one of them is bad (Failing=True) Reason=equals failing
Message:
```
"Unable to apply 1.0.0-abc: <identical to failing>

Also observed additional, non-problematic conditions:
* ...
* ...
```

It undoes the Clayton's "simplification" from 2018 but I think clarity beats simplification, especially when the simplified version is not helpful :)

Reply:
Well, we have an agreement on that the clarity is more helpful. The PR, as of the moment, would improve one aspect (the `Failing` condition) and worsen another (You can't get **all** the errors currently happening from the `ClusterVersion` resource). Which is something I don't like.

In the case we have an agreement that clarity is more helpful, I'll try to draft an implementation to list the errors from `MultipleErrors` (as they are and not simplified) instead of the `...an unknown error has occurred: MultipleErrors` message in the `Progressing` condition and keep the other non-problematic logic as is (we can iterate on improving the overall `Progressing` condition in new cards IMO) to something similar as:
```
"Unable to apply 1.0.0-abc: <identical to failing>

Also observed additional, non-problematic conditions:
* ...
* ...
```

Where `additional, non-problematic conditions` are `UpdateEffectNone` errors.

Reply:
> (You can't get all the errors currently happening from the ClusterVersion resource). Which is something I don't like.

There's a nuance: you can't see the less important "errors" (they are not actually errors) *only* when more important ones are happening. Which I think is still an improvement, because I think more people will be happy when they see a clearer message when something does not go well, than unhappy because they somehow need to see the less important message and they don't.

Reply:
As of now, the PR introduces an overall improvement and is sufficient by itself. In the future, we might rework the `Progressing` condition.